ansible/roles/dev/tasks/main.yml

```diff
@@ -91,16 +95,16 @@
   file:
     path: /var/lib/pulp
     state: directory
-    owner: apache
-    group: apache
+    owner: pulp
```

Comment:
I think this will do the trick:
s/pulp/{{ pulp_user }}/

If that works, I'm on board.